### PR TITLE
Added @yamada-ui/charts component info

### PR DESCRIPTION
**Pull request by hirotomoyamada**
## Description

Added @yamada-ui/charts component.

## Is this a breaking change (Yes/No):

No

### CODE REVIEW

File: contents/getting-started/frameworks/create-react-app.mdx [hunk 1]
```diff
@@ -49,6 +49,11 @@ If you want to use tables, calendars, carousels, etc., you need to install them
       description:
         "Provides a convenient dropzone component using [react-dropzone](https://react-dropzone.js.org/).",
     },
+    {
+      name: "@yamada-ui/charts",
+      description:
+        "Provides convenient chart components using　[recharts](https://recharts.org/)",
```
Comment (taroj1205) [inline contents/getting-started/frameworks/create-react-app.mdx:55]: Add a period at the end of the sentence.
The space before [recharts] should be half-width, not full-width.
```diff
-"Provides convenient chart components using　[recharts](https://recharts.org/)",
+"Provides convenient chart components using [recharts](https://recharts.org/).",

File: contents/getting-started/frameworks/gatsby.mdx [hunk 2]
```diff
@@ -49,6 +49,11 @@ If you want to use tables, calendars, carousels, etc., you need to install them
       description:
         "Provides convenient dropzone components using [react-dropzone](https://react-dropzone.js.org/).",
     },
+    {
+      name: "@yamada-ui/charts",
+      description:
+        "Provides convenient chart components using　[recharts](https://recharts.org/)",
```
Comment (taroj1205) [inline contents/getting-started/frameworks/gatsby.mdx:55]: Add a period at the end of the sentence.
The space before [recharts] should be half-width, not full-width.
```diff
-"Provides convenient chart components using　[recharts](https://recharts.org/)",
+"Provides convenient chart components using [recharts](https://recharts.org/).",

File: contents/getting-started/frameworks/hono.mdx [hunk 3]
```diff
@@ -69,6 +70,11 @@ If you want to use tables, calendars, carousels, etc., you need to install them
       description:
         "Provides a convenient dropzone component using [react-dropzone](https://react-dropzone.js.org/).",
     },
+    {
+      name: "@yamada-ui/charts",
+      description:
+        "Provides convenient chart components using　[recharts](https://recharts.org/)",
```
Comment (taroj1205) [inline contents/getting-started/frameworks/hono.mdx:76]: Add a period at the end of the sentence.
The space before [recharts] should be half-width, not full-width.
```diff
-"Provides convenient chart components using　[recharts](https://recharts.org/)",
+"Provides convenient chart components using [recharts](https://recharts.org/).",

File: contents/getting-started/frameworks/nextjs/app-router.mdx [hunk 4]
```diff
@@ -48,6 +48,11 @@ If you want to use tables, calendars, carousels, etc., you need to install them
       description:
         "Provides convenient dropzone components using [react-dropzone](https://react-dropzone.js.org/).",
     },
+    {
+      name: "@yamada-ui/charts",
+      description:
+        "Provides convenient chart components using　[recharts](https://recharts.org/)",
```
Comment (taroj1205) [inline contents/getting-started/frameworks/nextjs/app-router.mdx:54]: Add a period at the end of the sentence.
The space before [recharts] should be half-width, not full-width.
```diff
-"Provides convenient chart components using　[recharts](https://recharts.org/)",
+"Provides convenient chart components using [recharts](https://recharts.org/).",

File: contents/getting-started/frameworks/nextjs/index.mdx [hunk 5]
```diff
@@ -51,6 +51,11 @@ If you want to use tables, calendars, carousels, etc., you need to install them
       description:
         "Provides convenient dropzone components using [react-dropzone](https://react-dropzone.js.org/).",
     },
+    {
+      name: "@yamada-ui/charts",
+      description:
+        "Provides convenient chart components using　[recharts](https://recharts.org/)",
```
Comment (taroj1205) [inline contents/getting-started/frameworks/nextjs/index.mdx:57]: Add a period at the end of the sentence.
The space before [recharts] should be half-width, not full-width.
```diff
-"Provides convenient chart components using　[recharts](https://recharts.org/)",
+"Provides convenient chart components using [recharts](https://recharts.org/).",

File: contents/getting-started/frameworks/remix.mdx [hunk 6]
```diff
@@ -49,6 +49,11 @@ If you want to use tables, calendars, carousels, etc., you need to install them
       description:
         "Provides convenient dropzone components using [react-dropzone](https://react-dropzone.js.org/).",
     },
+    {
+      name: "@yamada-ui/charts",
+      description:
+        "Provides convenient chart components using　[recharts](https://recharts.org/)",
```
Comment (taroj1205) [inline contents/getting-started/frameworks/remix.mdx:55]: Add a period at the end of the sentence.
The space before [recharts] should be half-width, not full-width.
```diff
-"Provides convenient chart components using　[recharts](https://recharts.org/)",
+"Provides convenient chart components using [recharts](https://recharts.org/).",

File: contents/getting-started/frameworks/vite.mdx [hunk 7]
```diff
@@ -60,6 +60,11 @@ If you want to use tables, calendars, carousels, etc., you need to install them
       description:
         "Provides a convenient dropzone component using [react-dropzone](https://react-dropzone.js.org/).",
     },
+    {
+      name: "@yamada-ui/charts",
+      description:
+        "Provides convenient chart components using　[recharts](https://recharts.org/)",
```
Comment (taroj1205) [inline contents/getting-started/frameworks/vite.mdx:66]: Add a period at the end of the sentence.
The space before [recharts] should be half-width, not full-width.
```diff
-"Provides convenient chart components using　[recharts](https://recharts.org/)",
+"Provides convenient chart components using [recharts](https://recharts.org/).",

File: contents/getting-started/installation/index.mdx [hunk 8]
```diff
@@ -43,6 +43,11 @@ If you want to use tables, calendars, carousels, etc., you need to install them
       description:
         "Provides convenient dropzone components using [react-dropzone](https://react-dropzone.js.org/).",
     },
+    {
+      name: "@yamada-ui/charts",
+      description:
+        "Provides convenient chart components using　[recharts](https://recharts.org/)",
```
Comment (taroj1205) [inline contents/getting-started/installation/index.mdx:49]: Add a period at the end of the sentence.
The space before [recharts] should be half-width, not full-width.
```diff
-"Provides convenient chart components using　[recharts](https://recharts.org/)",
+"Provides convenient chart components using [recharts](https://recharts.org/).",

**Comment by hirotomoyamada** [comment]
@taroj1205 

Fixed typo.

**Review by taroj1205 — APPROVED** [review]
LGTM!

**Pull request merged**